stable/busybox/templates/deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
Use best practices for label.
https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/labels.md#standard-labels